config/initializers/httparty.rb

```diff
@@ -1,6 +1,4 @@
 # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
 require 'httparty'
-require 'httparty/logger/mailchimp_logger'
-require 'httparty/logger/full_contact_logger'
-::HTTParty::Logger.add_formatter(:mailchimp, HTTParty::Logger::MailchimpLogger)
-::HTTParty::Logger.add_formatter(:full_contact, HTTParty::Logger::FullContactLogger)
+::HTTParty::Logger.add_formatter(:mailchimp, ::Httparty::Logger::MailchimpLogger)
+::HTTParty::Logger.add_formatter(:full_contact, ::Httparty::Logger::FullContactLogger)
```

Comment:
I'm confused:
**::HTTP**arty::Logger.add_formatter(
  :full_contact, 
**::Http**arty::Logger::FullContactLogger)

The First invocation of the model is HTTParty... and the second is Httparty... huh?

Reply:
Yep. I tried a bunch of ways to get let them both be named HTTParty but the inflections just wouldn't work no matter what I tried. I figured it wasn't worth the headache to keep fixing.